themes/nekonight/nekonight-base.sh

```diff
@@ -0,0 +1,33 @@
+function _omb_theme_nekonight_git_prompt_info() {
+  local branch_name
+  branch_name=$(git symbolic-ref --short HEAD 2>&-)
+  local git_status=""
+
+  if [[ -n $branch_name ]]; then
+    git_status="${_omb_prompt_bold_white}(🐱 $branch_name $(_omb_theme_nekonight_scm_git_status))${_omb_prompt_normal}"
```

Comment:
Shouldn't this cat icon be switched in `nekonight-moon`? Please prepare a new `icon_*` variable and reference it here. Then, the new variable is set to be a proper value in each `nekonight.theme.sh` and `nekonight_moon.theme.sh`